Commands/StartCommand.php

```diff
@@ -32,7 +33,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $config = $this->initializeConfig($input, $output);
 
-        $handler = new ProcessManager($output, $config['port'], $config['host'], $config['workers']);
+        if (isset($config['factory'])) {
+            $factory = $config['factory'];
+            $handler = $factory::factory($output, $config['port'], $config['host'], $config['workers']);
```

Comment:
I'd go without static and just use `new $factory` or I would check the interface of $handler, to get better error messages when setBridge call below fail